infra/main.bicep

```
@@ -37,10 +37,10 @@ param formRecognizerResourceGroupLocation string = location
 
 param formRecognizerSkuName string = 'S0'
 
-param gptDeploymentName string = ''
+param gptDeploymentName string = '' // See main.parameters.json for default
```

```suggestion
param gptDeploymentName string
```

I think you'd want to remove this completely and make it a required parameter that is always set using the parameters file